apps/gateway-ui/src/App.tsx

```diff
@@ -93,10 +92,10 @@ export const App = React.memo(function Admin(): JSX.Element {
     <ApiProvider props={{ gateway }}>
       <Wrapper size='lg'>
         {error ? (
-          <VStack spacing={4}>
+          <Flex direction='column' gap={4}>
```

Comment:
The gaps of things that were formerly `VStack` need to be increased, since `VStack` provided its own additional `margin`-based gaps.
```suggestion
          <Flex direction='column' gap={6}>
```

Reply:
Thanks got it. Will update the code accordingly!